src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -95,9 +93,8 @@ public ProjectToUpdate(string filePath, bool allowAutoRestore, ProjectIdInfo pro
             _queue = new BufferBlock<ProjectToUpdate>();
             _processLoopCancellation = new CancellationTokenSource();
             _processLoopTask = Task.Run(() => ProcessLoopAsync(_processLoopCancellation.Token));
-            _assemblyLoader = assemblyLoader;
+            _analyzerAssemblyLoader = assemblyLoader;
```

Comment:
Maybe rename `assemblyLoader` to `analyzerAssemblyLoader` while we are renaming.